### PR TITLE
info command output now ordered

### PR DESCRIPTION
**Pull request by bahmatovich**
issue #3

### CODE REVIEW

**Comment by secwall** [comment]
Unfortunately this is not the fix of original problem.
Example:
```
d = {'32': {}, '1': {}, '12': {}, '3': {}}
orderedret = OrderedDict()
for version in sorted(d):
    orderedret[version] = d[version]
print(json.dumps(orderedret))
```
Output is 
> {"1": {}, "12": {}, "3": {}, "32": {}}

**Comment by bahmatovich** [comment]
Ok, i will cast it as int

**Comment by secwall** [comment]
Could you check if [this patch](https://gist.github.com/secwall/ad08d837c8d9898cd052eab59ba57109) solves original problem?

File: pgmigrate.py [hunk 1]
```diff
@@ -198,7 +198,7 @@ def _get_info(base_dir, baseline_v, target_v, cursor):
     '''
     ret = {}
     cursor.execute('SELECT ' + ', '.join(REF_COLUMNS) +
-                   ' from public.schema_version;')
+                   ' from public.schema_version order by version;')
```
Comment (secwall) [inline pgmigrate.py]: Do we really need order by here?

File: pgmigrate.py [hunk 2]
```diff
@@ -219,7 +219,11 @@ def _get_info(base_dir, baseline_v, target_v, cursor):
             if num not in ret:
                 ret[num] = migrations_info[version]
 
-    return ret
+    orderedret = OrderedDict()
+    for version in sorted(ret, key=int):
+        orderedret[version] = ret[version]
+
+    return orderedret
```
Comment (secwall) [inline pgmigrate.py:226]: May be we just need to sort state just before output? Sorted migrations info is not required in other functions.

**Review by secwall — COMMENTED** [review]
I like the idea in general. But it seems that some improvements are possible.

File: pgmigrate.py (hunk 2 above)
Comment (bahmatovich) [reply pgmigrate.py:226]: Yes, you are right.

**Comment by secwall** [comment]
I've commited [patch](https://gist.github.com/secwall/ad08d837c8d9898cd052eab59ba57109) as  77d7df4. So I close this.

**Pull request closed without merging**